src/averageAges.js

```diff
@@ -15,11 +16,15 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  const men = century
+    ? people.filter((p) => p.sex === 'm' && Math.ceil(p.died / 100) === century)
```

Comment:
- there is no need to use parentheses for `p` arg if you write a single-line method;
- or u can use object destructuring here: `({ sex, died })`;
- it's better to give full names for args inside array methods; it will be better to understand your code and much easier to refactor in the future (e.g. `person` instead of simple `p`).